VKApp/VKApp/TabBar/First/PhotosCollectionViewController.swift

```diff
@@ -2,22 +2,28 @@
 // Copyright © RoadMap. All rights reserved.
 
 import Alamofire
+import RealmSwift
 import UIKit
 
 /// Экран фотографий пользователя
 final class PhotosCollectionViewController: UICollectionViewController {
     // MARK: - Private properties
 
     private let networkService = NetworkService()
+    private let realmService = RealmService()
+    private let photoResult = Size()
 
+    private var notificationToken: NotificationToken?
     private var photosUrlPath: [String] = []
+    private var photos: [Size] = []
```

Comment:
```suggestion
    private var  sizes: [Size] = []
```
имя и объект разные
либо photos: [Photo]